PencilCase/PencilCase/scripts/CustomBindings/SelectChangeBinding.ts

```diff
@@ -0,0 +1,14 @@
+﻿class SelectChangeBinding implements KnockoutBindingHandler {
+    public init(element: any, valueAccessor: () => any, allowBindingAccessor: KnockoutAllBindingsAccessor, viewModel: any, bindingContext: KnockoutBindingContext) {
```

Comment:
For select component, if you change select option on UI, the binding field in VM will change, it makes sense.
But if you update the filed in VM, the UI won't be updated. This is binding handler here is used to handle such scenarios. 
When VM field changes, update UI.